pkg/monitors/statsd/parser.go

```diff
@@ -37,12 +37,15 @@ func parseDogstatsdTags(s string) (string, map[string]string) {
 		s = s[:tagsIdx]
 		dims = make(map[string]string)
 		for _, t := range strings.Split(tagsRaw, ",") {
-			parts := strings.Split(t, ":")
+			parts := strings.SplitAfterN(t, ":", 2)
```

Comment:
Why not just `strings.SplitN(t, ":", 2)`?  It seems like that would eliminate the need to do the trim below?